__tests__/__snapshots__/ord.test.js.snap

```diff
@@ -174,7 +174,7 @@ exports[`Tests for default ORD document Successfully create ORD Documents with d
   "packages": [
     {
       "description": "Description for capire bookshop ord sample",
-      "ordId": "capirebookshopordsample:package:undefined:v1",
+      "ordId": "capirebookshopordsample:package:capire.bookshopordsample:v1",
```

Comment:
Hi @Fannon, is this package id valid? `<namespace>:<conceptName>:<resourceName>:[v<majorVersion>]`, namesapce and resourceName are reverted?

Reply:
No, this is not valid. The first fragment is not an ORD namespace. It should have the same namespace as you define in the cdsrc file. 

The third fragment (resource name) looks fine. This is just a fully qualified name from CAP view, not necessarily a namespace.

Reply:
```suggestion
      "ordId": "capire.bookshopordsample:package:capirebookshopordsample:v1",
```

This is more likely the correct ORD ID.